docs/API/Users_Guide/Data_Analysis.md

```diff
@@ -150,6 +153,198 @@ gene_start      gene_end        symbol  id
 (truncated)
 ```
 
+## Gene Expression Examples
+
+### Gene Expression Availability Endpoint
+
+The purpose of this endpoint is to retrieve the availability of gene expression data for only cases, genes, or both. The availability response informs the user if gene expression data exists for each case or gene, which are specified with case and gene ids. Gene expression data is only available for protein-coding genes.
+
+__Example 1__: A user wants to get the availability of gene expression data for a set of cases and genes.
+
+```Filter
+{
+  "case_ids": [
+    "6d4f38db-a97b-4dc0-8dc5-2ac7f2cc5e38",
+    "e3b32485-b204-43a7-93a5-601408fcdf96"
+  ],
+  "gene_ids": [
+    "ENSG00000141510",
+    "ENSG00000181143"
+  ]
+}
+```
+
+```Shell
+curl -X 'POST' \
+  'https://api.gdc.cancer.gov/gene_expression/availability' \
+  -H 'accept: application/json' \
+  -H 'Content-Type: application/json' \
+  -d '{
+  "case_ids": [
+    "6d4f38db-a97b-4dc0-8dc5-2ac7f2cc5e38",
+    "e3b32485-b204-43a7-93a5-601408fcdf96"
+  ],
+  "gene_ids": [
+    "ENSG00000141510",
+    "ENSG00000181143"
+  ]
+}'
+```
+
+```Response
+{
+    "cases": {
+        "details": [
+            {
+                "case_id": "6d4f38db-a97b-4dc0-8dc5-2ac7f2cc5e38",
+                "has_gene_expression_values": false
+            },
+            {
+                "case_id": "e3b32485-b204-43a7-93a5-601408fcdf96",
+                "has_gene_expression_values": true
+            }
+        ],
+        "with_gene_expression_count": 1,
+        "without_gene_expression_count": 1
+    },
+    "genes": {
+        "details": [
+            {
+                "gene_id": "ENSG00000141510",
+                "has_gene_expression_values": true
+            },
+            {
+                "gene_id": "ENSG00000181143",
+                "has_gene_expression_values": true
+            }
+        ],
+        "with_gene_expression_count": 2,
+        "without_gene_expression_count": 0
+    }
+}
+```
+
+### Gene Expression Values Endpoint
+
+The purpose of this endpoint is to retrieve the gene expression values for the given cases and genes. The response is a TSV containing the expression values for genes to cases.
+
+__Example 1__: A user wants to get expression values using case ids and gene ids.
+
+```Filter
+{
+    "case_ids": [
+        "6d4f38db-a97b-4dc0-8dc5-2ac7f2cc5e38",
+        "e3b32485-b204-43a7-93a5-601408fcdf96",
+        "000ead0d-abf5-4606-be04-1ea31b999840",
+        "001ab32d-f924-4753-ad67-4366fb845ae6"
+    ],
+    "gene_ids": [
+        "ENSG00000141510",
+        "ENSG00000181143"
+    ],
+    "unit": "median-centered"
```

Comment:
```suggestion
    "unit": "median-centered",
    "format": "tsv"
```
Let's add the tsv format parameter to be explicit.

Reply:
added